website/www/site/content/en/blog/splittable-do-fn-is-available.md

```diff
@@ -0,0 +1,58 @@
+---
+title:  "Splittable DoFn in Apache Beam is Ready to Use"
+date:   2020-12-16 00:00:01 -0800
+categories:
+  - blog
+aliases:
+  - /blog/2020/12/16/splittable-do-fn-is-available.html
+authors:
+  - boyuanzz
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+In 2017, [Splittable DoFn Blog Post](https://beam.apache.org/blog/splittable-do-fn/) proposed
+to build [Splittable DoFn](https://s.apache.org/splittable-do-fn) APIs as the new recommended way of
+building I/O connectors. Now Splittable DoFn is ready to use. Try out building your Splittable DoFn
```

Comment:
Summarize the goals of SDF, romanticize it, and tell users why they should be excited to use the new feature. Some questions you should answer here:
 
  * Why should I prefer SDF over the 'old' I/O connectors? (new capabilities? better perf? cleaner api?)
  * Is it worth migrating my existing i/o connector to SDF? Why?
  * What's next for SDFs? (optional)
  * Why did it take 3 years to do this? (optional)
  * Is there a specific place/component/label to report bugs? (optional)

These details may duplicate content from the other blog posts but that is fine. A high level, short summary, of them would help me avoid having to sift through the links.

Reply:
Just updated the blog with more details. Would you like to take another look?